code/src/components/Todo.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import moment from 'moment'
+import styled from 'styled-components'
+
+import { todos } from 'reducers/todos'
+import { Button } from 'library/Buttons'
+import { CustomCheckbox } from 'components/CustomCheckbox'
+import { TimeIcon } from 'library/Icons'
+
+export const Todo = (props) => {
+  const { id, text, dueDate, complete, startDate } = props.item;
+  const dispatch = useDispatch()
+
+  //checkbox click dispatches a toggle completed action to Redux:
+  const handleCheckboxClick = () => {
+    dispatch(todos.actions.toggleCompleted(id))
+  }
+
+  const handleRemoveButtonClick = () => {
+    dispatch(todos.actions.removeTodo(id))
+  }
+
+  return (
+    <ListItem>
+      <ItemLineWrapper> 
+        <ItemTextWrapper>
+          <label>
+            <CustomCheckbox
+              isChecked={complete} 
+              onChangeHandler={handleCheckboxClick}/>
+          </label>
+          <TextLine checked={complete}>{text}</TextLine>
+        </ItemTextWrapper>
+        <Button 
+          type='button' 
+          onClick={handleRemoveButtonClick} 
+          background='#d3947a'
+        >
+          Remove
+        </Button>
+      </ItemLineWrapper>
+      {/* Note to self: Implement functionality so that text turns red if duedate is passed */}
+      <ItemLineWrapper> 
+        <TimeLine> <TimeIcon src="../assets/clock.svg" /> {moment(startDate).fromNow()}</TimeLine>
```

Comment:
The `startDate` property does not exist on the todos so this will always say "a few seconds ago"